openfisca_country_template/variables/demographics.py

```diff
@@ -1,17 +1,26 @@
 # -*- coding: utf-8 -*-
 
-# This file defines the variables of our legislation.
-# A variable is property of a person, or an entity (e.g. a household).
+# This file defines variables for the modelled legislation.
+# A variable is a property of an Entity such as a Person, a Family…
 # See http://openfisca.org/doc/variables.html
 
-# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+# Import from openfisca-core the common Python objects used to code the legislation in OpenFisca
 from openfisca_core.model_api import *
-# Import the entities specifically defined for this tax and benefit system
+# Import the Entities specifically defined for this tax and benefit system
 from openfisca_country_template.entities import *
-
+# Import additional NumPy helpers to work on specific data types
 from numpy import datetime64
```

Comment:
I don't think we are actually using this import anymore, and that's why Flake is complaning.

Reply:
I agree, but I wanted a double check from the person who wrote the initial code :wink: